### ISSUE TITLE
Add "maintainers" field to package.json

### ISSUE THREAD

**Issue by dkerr64** [oznu/homebridge-config-ui-x#2076]
### Describe The Bug

As #2070 has been closed, opening a new issue to track the need to add "maintainers" field to package.json...

the UI 4.56.2 and below are not detecting that there is a new UI release. To test, I just installed hombridge following instructions [here](https://github.com/homebridge/homebridge/wiki/Install-Homebridge-on-Debian-or-Ubuntu-Linux). And it installed UI v4.56.2 and reports that it is up-to-date...
![image](https://github.com/homebridge/homebridge-config-ui-x/assets/17837612/8b7aff89-7a1c-4295-b7dc-8278a35e80d7)

![image](https://github.com/homebridge/homebridge-config-ui-x/assets/17837612/9f05d663-d263-4830-b0e7-441a39076e0a)
So your average user has no idea that the Homebridge UI is out-of-date, AND will not get any prompting for ANY other plugin that has had an update published.... unless the plugin owner adds the maintainers list to their package.json.

This UI needs to add it to its own package.json, which will hopefully then cause homebridge to highlight that a new UI is available.

### Homebridge UI Version

4.56.2

### Homebridge Version

1.8.3

### Node.js Version

20.14

### Operating System

Ubuntu / Debian (or a variant)

### Environment Info

None

### Raspberry Pi Model

None

**Comment by mkz212** [comment]
👍

**Comment by mkz212** [comment]
`"maintainers": ["maintainer-name"]` to the package.json - that's all !

**Comment by NorthernMan54** [comment]
Duplicate of #2077

**Comment by dkerr64** [comment]
> Duplicate of #2077

No, not really... this is documenting a specific change that needs to be made to work-around a bug introduced by NPM. 

#2077 is a duplicate of #2070 which address a bug in homebridge-config-ui-x (that attempted to dereference a object member without first testing if that object member exists).

Newer issues are normally closed as a dup of an earlier issue, not the other way around.

**Comment by dkerr64** [comment]
@NorthernMan54 @mkz212 @donavanbecker Can I respectfully ask that this issue be reopened.  It is not fixed and I do not see a fix queued up in the beta-4.56.4 branch... unless you are intending not to fix this, and instead wait on NPM.  In which case come out and say that.

I'm not sure why folks are so eager to close valid issues opened to document a necessary change.

**Comment by donavanbecker** [comment]
@dkerr64 Updating to v4.56.3 does resolve this. So this is resolved.

**Comment by dkerr64** [comment]
> @dkerr64 Updating to v4.56.3 does resolve this. So this is resolved.

No it doesn't.  Maybe I need to restate the problem...

Anyone with 4.56.2 or less will not see any indication that there is a new version of the UI available to install.  So yes, the problem is solved in 4.56.3, but the average user is not ever going to install that... they will forever remain on 4.56.2.

The reason is that 4.56.2 throws exception because "maintainers" does not exist, so the UI never shows "update available" so to work around this problem, packages published to npmjs.com must include a maintainers array in the package.json.  So that 4.56.2 will not throw an exception when it is missing.  This should then restore the notice that an update is available.

Right?
